README.md

```diff
@@ -31,22 +31,34 @@ in the top-level object of the document via the `attribution` field as defined i
 
 Links to a [OGC Web Map Tile Service](https://www.ogc.org/standards/wmts) (WMTS) implementation (versions 1.x).
 
-| Field Name      | Type                 | Description |
-| --------------- | -------------------- | ----------- |
-| rel             | string               | **REQUIRED**. Must be set to `wmts`. |
-| href            | string               | **REQUIRED**. Link to the WMTS, without any WMTS specific query parameters. |
-| wmts:layer      | string\|\[string]    | **REQUIRED**. The layers to show on the map, either a list of layer names or a single layer name. |
+| Field Name      | Type                 | Description                                                                                                      |
+| --------------- | -------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| rel             | string               | **REQUIRED**. Must be set to `wmts`.                                                                             |
+| href            | string               | **REQUIRED**. Link to the WMTS, without any WMTS specific query parameters.                                      |
+| wmts:layer      | string\|\[string]    | **REQUIRED**. The layers to show on the map, either a list of layer names or a single layer name.                |
```

Comment:
why is this required? layer can be enclosed in the link.

Reply:
Because the link doesn't contain parameters.